Rakefile

```
@@ -173,12 +173,26 @@ rescue *HTTP_ERRORS => e
   "Error #{e.inspect}"
 end
 
-def create_post_file!(filename, response, date, host)
-  title = parse_title(response['title'])
-  components = parse_valid_components(response['labels'])
+def get_nonempty_components(gh_labels)
+  # Parses the GitHub labels, and requires user input if no valid components were found
+  components = parse_valid_components(gh_labels)
   if components.empty?
-    puts "No label assigned to the PR yet; you will need to add one manually from #{COMPONENTS}"
+    puts "No label assigned to the PR yet; you will need to add one or more (comma-separated) manually from #{COMPONENTS}"
+    while true
+      components_input = gets.split(',').map(&:strip).map(&:downcase).uniq
```

The labels are printed out with double quotes, so I included double quotes when passing my labels, which caused it to not recognize the label. I think it would be better to strip quotes so that it doesn't matter if the user uses quotes or not:

```suggestion
      components_input = gets.gsub(/['"]/, '').split(',').map(&:strip).map(&:downcase).uniq
```

---

Makes sense, added your suggestion, thanks!